tests/python/unittest/test_operator.py

```diff
@@ -2396,21 +2396,22 @@ def check_l2_normalization(in_shape, mode, norm_eps=1e-10):
     exe = out.simple_bind(ctx=ctx, data=in_data.shape)
     output = exe.forward(is_train=True, data=in_data)
     # compare numpy + mxnet
-    assert_almost_equal(exe.outputs[0].asnumpy(), np_out, rtol=1e-5)
+    assert_almost_equal(exe.outputs[0].asnumpy(), np_out, rtol=1e-2 if dtype is 'float16' else 1e-5)
```

Comment:
can  you also pass atol here. Default is 1e-20 which may result in test becoming flaky if the numbers are small.

Reply:
Done